bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/RowResultAdapter.java

```diff
@@ -179,7 +186,19 @@ public void finishCell() {
               TimestampConverter.bigtable2hbase(this.timestamp),
               this.value,
               this.labels);
-      this.currentFamilyCells.add(rowCell);
+
```

Comment:
please add a comment referring readers to the class javadoc for explaining this:

`// dedupe user visible cells. Please see class javadoc for details`